gcc/rust/lex/rust-lex.h

```diff
@@ -139,7 +139,8 @@ class Lexer
 
 public:
   // Construct lexer with input file and filename provided
-  Lexer (const char *filename, RAIIFile input, Linemap *linemap);
+  Lexer (const char *filename, RAIIFile input, Linemap *linemap,
+	 bool dump_lex_enabled);
```

Comment:
```suggestion
	 bool dump_lex_enabled = false);
```

This allows lexer users who don't care about the dumping to not pass an extra parameter (macros, modules, etc)

Reply:
fixed